gke/cluster.tf

```diff
@@ -14,9 +14,25 @@
 # limitations under the License.
 ############################################################################################
 
+// Project API Services
+
+resource "google_project_service" "compute" {
+  service = "compute.googleapis.com"
+}
```

Comment:
The wiki page doesn't say enabling the Compute API is required.  Is this necessary?

Reply:
yes, the compute API is a requirement for the container API, so both are needed.